### PR TITLE
Improve Visual Studio support - documentation part

### PR DESCRIPTION
**Pull request by fabienrohrer**
https://github.com/omichel/webots/pull/4223

### CODE REVIEW

**Comment by fabienrohrer** [comment]
👍

File: guide/using-visual-cpp-with-webots.md [hunk 1]
```diff
@@ -142,25 +142,40 @@ Now the controller should pause when it reaches one of your breakpoints.
...
-The corresponding precompiled libraries are incompatible with Visual C++,
-because they are compiled using the `gcc` compiler tool chain embedded in Webots.
+`youBot library`, etc.).
 
-To use these libraries with your project, their source files should be compiled directly in your project,
-exactly as it's done for the `libCppController` library (cf. instructions above).
+The precompiled `C` libraries are released with the corresponding Visual Studio `.lib` file
+(the linker to the `.dll` file) to facilitate their integration into a Visual Studio project.
+There is one `.lib` file per architecture (`.x32.lib` for the 32 bits architecture
+and `.x64.lib` for the 64 bits architecture).
```
Comment (omichel) [inline guide/using-visual-cpp-with-webots.md]: I wouldn't name these linkers this way. I would follow the same template as the gcc `.a` linkers, e.g., store a `.lib` file in the correct directory, as it is currently the case for `Controller.lib`:
```
msys64/mingw32/lib/Controller.lib # 32 bit version
msys64/mingw64/lib/Controller.lib # 64 bit version
```

Comment (fabienrohrer) [inline guide/using-visual-cpp-with-webots.md]: So if I understand well, you propose to move the following files there:

```
# 32 bit version
msys64/mingw32/lib/car.lib
msys64/mingw32/lib/driver.lib
msys64/mingw32/lib/youbot_control.lib
# 64 bit version
msys64/mingw64/lib/car.lib
msys64/mingw64/lib/driver.lib
msys64/mingw64/lib/youbot_control.lib
```

Comment (omichel) [inline guide/using-visual-cpp-with-webots.md]: Yes, exactly. This is more aligned with what we have been doing so far with Controller.lib and it follows the msys64 files organization rules.

Comment (fabienrohrer) [inline guide/using-visual-cpp-with-webots.md]: Ok, I modifying this.

**Comment by fabienrohrer** [comment]
👍

**Comment by omichel** [comment]
:+1:

**Comment by fabienrohrer** [comment]
👍

**Pull request merged**